products/compute/api.yaml

```diff
@@ -9871,11 +9871,11 @@ objects:
                           - :TEMPORARY_REDIRECT
                       - !ruby/object:Api::Type::Boolean
                         name: 'stripQuery'
-                        default_value: false
+                        required: true
```

Comment:
Idea: does it make sense to do these as overrides in terraform.yaml instead? I don't think inspec/ansible have the same constraints around non-empty blocks, and so putting it there would make it easier to call out why we're doing this (we could group them all together more easily)

Reply:
That makes sense. Do you think it is a good idea to explicitly say in the description "This is only required to ensure we do not allow an empty block in a terraform config"?

Reply:
Sure!